### DISCUSSION TITLE
YITH WooCommerce Request a Quote

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5273] [New Feature/Ideas Requests]
Integrate Fluxstore with [https://tr.wordpress.org/plugins/yith-woocommerce-request-a-quote/](https://tr.wordpress.org/plugins/yith-woocommerce-request-a-quote/)

**Comment by officialsuyogdixit** [comment]
This feature is a must.

**Reply by osman-kocak** [reply]
@minhcasi

**Comment by arif07159** [comment]
Really its required...
